test/e2e/specs/editor/blocks/navigation.spec.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * WordPress dependencies
+ */
+const { test, expect } = require( '@wordpress/e2e-test-utils-playwright' );
+
+test.describe(
+	'As a user I want the navigation block to fallback to the best possible default',
+	() => {
+		test.beforeAll( async ( { requestUtils } ) => {
+			await requestUtils.activateTheme( 'emptytheme' );
+		} );
+
+		test.beforeEach( async ( { admin } ) => {
+			await admin.createNewPost();
+		} );
+
+		test.afterAll( async ( { requestUtils } ) => {
+			await requestUtils.activateTheme( 'twentytwentyone' );
+		} );
+
+		test.afterEach( async ( { requestUtils } ) => {
+			await requestUtils.deleteAllPosts();
+		} );
+
+		test( 'default to a list of pages if there are no menus', async ( {
+			editor,
+			page,
+		} ) => {
+			await editor.insertBlock( { name: 'core/navigation' } );
+
+			// Necessary to wait for the block to be in loaded state else
+			// initial rendered state will not be resolved.
+			await page.waitForLoadState( 'networkidle' );
+
+			// Check Page List is in the list view.
+
+			// Open the list view.
+			await page.click( '[aria-label="Document Overview"i]' );
+
+			const listView = await page.locator(
+				`//table[contains(@aria-label,'Block navigation structure')]`
+			);
+
+			// Click the Navigation block expander within the List View.
```

Comment:
I don't think we need to test this. It's a test for the list view, not a test for the navigation block.

Reply:
The reason we do this is to have a generic way to check that page list is "visible" to the user.

Alternatives

- remove entirely and rely on non-user perceivable things like the snapshot of the code (see end of test)
- assert on presence of block(s) within the editor _canvas_